lib/bloc/backup/backup_bloc.dart

```diff
@@ -87,6 +89,19 @@ class BackupBloc {
   final _backupActionsController = StreamController<AsyncAction>.broadcast();
   Sink<AsyncAction> get backupActionsSink => _backupActionsController.sink;
 
+  Stream<bool> get promptBackupSubscription => Rx.combineLatestList([
+        backupSettingsStream,
+        promptBackupStream,
+        promptBackupDismissedStream,
+      ]).where((list) {
+        final settings = list[0] as BackupSettings;
+        final dismissed = list[2] as bool;
+        return settings.promptOnError && !dismissed;
```

Comment:
NIT: We can add `&& needSignIn` here and get rid of the map.

Reply:
Not possible, doing that, if `needSignIn` is false we are not emitting the event.

We need to receive the event whenever `needSignIn` is true or false; the rules to filter are the promptOnError and not dismissed.